test/cli/pull/ok-allow-env-target/in/.keboola/manifest.json

```diff
@@ -18,7 +18,7 @@
     "dataAppConfig": "app/{component_id}/{config_name}"
   },
   "allowedBranches": [
-    "__main__"
+    "2"
   ],
   "ignoredComponents": [],
   "templates": {
```

Comment:
Improved tests, `2` is ID of a branch. It should be adhoc replaced by the value from ENV, ... it works already works for other occurences of the `BranchID(2)`, and now also for the `AllowedBranch("2")` definition.